tests/Tests/500__jupyterhub/jupyterhub-user-access.robot

```diff
@@ -58,3 +62,19 @@ Test User In JH Users Group
       Launch JupyterHub From RHODS Dashboard Dropdown
     END
     Login Verify Access Level  ldap-user1  ${TEST_USER.PASSWORD}  ${AUTH_TYPE}  user
+
+Verify User Can Set Custom RHODS Groups
+    [Documentation]    Tests the JH access level when using custom rhods groups
+    ...                different from rhods-admins and rhods-users
+    [Tags]  Sanity
+    ...     ODS-293
+    [Setup]      Set Standard RHODS Groups Variables
+    Open OCP Console
+    Login To OCP
+    Create Custom Groups
+    Add Test Users To Custom Groups
+    Remove Test Users From RHODS Standard Groups
+    Apply New Groups Config Map
+    Rollout JupyterHub
+    Check New Access Configuration Works As Expected
+    [Teardown]   Restore Standard Configuration
```

Comment:
`Restore Standard RHODS Groups Configuration` seems more specific

Reply:
yep, agree. i'm changing it

Reply:
done